scale/recipe/test/test_models.py

```diff
@@ -543,6 +543,10 @@ def test_successful_supersede_different_recipe_type(self):
         recipe = Recipe.objects.get(id=handler.recipe_id)
         recipe_job_1 = RecipeJob.objects.select_related('job').get(recipe_id=handler.recipe_id, job_name='Job 1')
         recipe_job_2 = RecipeJob.objects.select_related('job').get(recipe_id=handler.recipe_id, job_name='Job 2')
+        job_exe_2 = job_test_utils.create_job_exe(job=recipe_job_2.job)
+        from product.models import ProductFile
+        from product.test import utils as product_test_utils
```

Comment:
This block should be in a try/except if you want to maintain the package hierarchy where ProductFile is optional here.